graphql/src/test/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilderTest.java

```diff
@@ -130,13 +131,8 @@ void successful() throws Exception {
         final File graphqlSchemaFile =
                 new File(getClass().getResource("/test.graphqls").toURI());
         final GraphqlServiceBuilder builder = new GraphqlServiceBuilder();
-        final DataLoader<String, String> dataLoader =
-                DataLoaderFactory.newDataLoader(keys -> CompletableFuture.supplyAsync(() -> keys));
         final GraphqlService service = builder.schemaFile(graphqlSchemaFile)
                                               .instrumentation(SimpleInstrumentation.INSTANCE)
-                                              .configureDataLoaderRegistry(dlr -> {
-                                                  dlr.register("dummy1", dataLoader);
-                                              })
```

Comment:
The new API is only tested with builder validations. Can we test if the new API performs correctly?

Reply:
Added API test 😄